recipes/sagemaker-pytorch-container/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "sagemaker-pytorch-container" %}
+{% set version = "1.0.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/aws/{{ name }}/archive/v{{ version }}.tar.gz"
+  sha256: f211f32ccc299714d3e6a386e7f7dbea71033d82246ad45055ea7da966f57721
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win or py==37]
```

Comment:
```suggestion
  noarch: python
```

Reply:
Unskipping Windows will not work due to the sagemaker_containers dependency only being capable of running on linux/osx

Reply:
True, but we can ignore windows in staged-recipes and the feedstock will be built on linux only in the feedstock as the recipe is noarch: python. If `sagemaker_containers` package gets a windows package in the future this package will be installable. Otherwise the user will get an error if they tried to install this package on windows.

Reply:
Got it. Just to be clear, we can still get this merged with windows failing?

Reply:
> Just to be clear, we can still get this merged with windows failing?

Yes

> Also I am curious why this would be beneficial over including a windows selector?

Noarch python packages are built only on linux and works on all platforms across all supported python versions. So, you don't have to rebuild the package for a new architecture nor for a new python version.

Reply:
@isuruf Thanks, I understand now. Your commits have been added.